src/components/StaffPayment/StaffPayment.vue

```diff
@@ -230,10 +233,10 @@ export default class StaffPayment extends Vue {
   @Emit('update:staffPaymentData')
   private emitStaffPaymentData ({
     option = this.staffPaymentData.option,
-    routingSlipNumber = this.staffPaymentData.routingSlipNumber || '',
-    bcolAccountNumber = this.staffPaymentData.bcolAccountNumber || '',
-    datNumber = this.staffPaymentData.datNumber || '',
-    folioNumber = this.staffPaymentData.folioNumber || '',
+    routingSlipNumber = this.staffPaymentData.routingSlipNumber?.trim() || '',
+    bcolAccountNumber = this.staffPaymentData.bcolAccountNumber?.trim() || '',
+    datNumber = this.staffPaymentData.datNumber?.trim() || '',
+    folioNumber = this.staffPaymentData.folioNumber?.trim() || '',
```

Comment:
Why do you need to do both? Doesn't `v-model.trim` cause the variable to be trimmed?

Conversely, if the `@input` calls emitStaffData() on every change, that should cause the state variables to update, which the v-text-field should pick up, right?

Reply:
see above, I was noticing that just one was not enough ^